requirements/pip.txt

```diff
@@ -112,7 +112,7 @@ django-debug-toolbar==3.2.4
 # For enabling content-security-policy
 django-csp==3.7
 # For setting the permissions-policy security header
-django-permissions-policy==4.7.0
+django-permissions-policy==4.8.0
```

Comment:
```suggestion
# We can not upgrade to 4.8.0 because it removes `interest-cohort` feature and we are still using it.
django-permissions-policy==4.7.0
```

@davidfischer is `interest-cohort` something we want to keep using?

Reply:
It mostly doesn't matter anymore. Google abandoned it.